Example/Segmentify-Demo/Controller/HomeViewController.swift

```diff
@@ -33,11 +34,23 @@ class HomeViewController: UIViewController {
         // no large title
         navigationController?.navigationBar.prefersLargeTitles = false
         // pageView Request
-        sendPageViewRequest()
+        //sendPageViewRequest()
+        sendSearchPageViewRequest()
         // update table view
         tableView.reloadData()
     }
     
+    func sendSearchPageViewRequest() {
+        let pageViewObj = SearchPageModel()
+        pageViewObj.query = ""
+        pageViewObj.lang = "TR"
+        SegmentifyManager.sharedManager().sendSearchPageView(segmentifyObject: pageViewObj){
+            (response: SearchModel) in
+            self.response = response
+        }
+        //SegmentifyManager.sharedManager().sendSearchClickView(instanceId: "SEARCH", interactionId: "1")
```

Comment:
do we need this line?

Reply:
@ackepenek  I added this to the guide so I can delete it.